backend/api/Database/Models/PlannedTask.cs

```diff
@@ -22,6 +22,9 @@ public class PlannedTask
 
         [Required]
         public Position TagPosition { get; set; }
+        [Required]
+        public Pose Pose { get; set; }
```

Comment:
RobotPose instead to distinguish between tag and robot position?

Reply:
Tag would not have a Pose, it is a Position

Reply:
So remove the Pose from the PlannedTask then?

Reply:
No, it's there as the target robot pose for that task